code/modules/reagents/chemistry/reagents/medicine.dm

```diff
@@ -644,6 +648,21 @@
 					M.adjustBruteLoss(rand(0,15))
 					M.adjustToxLoss(rand(0,15))
 					M.adjustFireLoss(rand(0,15))
+					if(ishuman(M) && M.mind && prob(10)) // To prevent people from using this on monkies to cause pandemics
+						var/mob/living/carbon/human/H = M
+						// Their body is being suffused with magical life-inducing stuff - so other things get lively, too
+						// Things are going to get interesting for the reviving party, too
+						var/virus_type = pick(list(
+							/datum/disease/advance/flu,
+							/datum/disease/advance/cold,
+							/datum/disease/brainrot,
+							/datum/disease/magnitis,
+							/datum/disease/fake_gbs, // honk
+							/datum/disease/cold9
+						))
+						var/datum/disease/D = new virus_type()
+						H.AddDisease(D)
```

Comment:
yes they should

There are numerous means of reviving people, and each should have their own costs, consequences, and deliberations, to make death a non-trivial obstacle to medical staff. Any situation that SR can be used, for example, a body transplant and a defib can also be used - at the cost of losing the original body's "appearance", for the unclonable species.

The disease causing can be mitigated by medical anticipating this, and placing the patient into quarantine for a short period while they scout out that the patient is either clean, or needs treatment before a pandemic spreads.

The grief potential is mitigated by both the requirement for a sentient player, and a cap of 3 max chances at spawning a disease per person (before requiring a cryotrip, which is extremely non-subtle) ((100% - 90%^3) = 27.1% chance on a given person)